addons/survey/views/survey_templates_statistics.xml

```diff
@@ -331,11 +344,28 @@
-                                        <t t-if="question.question_type == 'datetime'">
-                                            <span t-if="no_print_url" t-field="input_line.value_datetime"/>
-                                            <a t-else="" t-att-href="input_line.user_input_id.get_print_url()" t-field="input_line.value_datetime"></a>
+                                        <!-- If answer already covered by filter or if there is only one line to display, do not allow filtering on it again -->
+                                        <t t-if="question.id in disabledQuestionsFilters or len(table_data) == 1">
+                                            <span t-field="input_line.value_numerical_box" t-if="question.question_type == 'numerical_box'"/>
+                                            <span t-field="input_line.value_date" t-if="question.question_type == 'date'"/>
+                                            <span t-field="input_line.value_datetime" t-if="question.question_type == 'datetime'"/>
+                                        </t>
+                                        <t t-else="">
+                                            <a t-field="input_line.value_numerical_box" t-if="question.question_type == 'numerical_box'"
+                                                class="text-primary filter-add-answer"
+                                                data-model="L" t-att-data-answer-id="input_line.id"
+                                                role="button" aria-label="Filter surveys" title="Only show survey results having selected this answer"/>
+                                            <a t-field="input_line.value_date" t-if="question.question_type == 'date'"
+                                                class="text-primary filter-add-answer"
+                                                data-model="L" t-att-data-answer-id="input_line.id"
+                                                role="button" aria-label="Filter surveys" title="Only show survey results having selected this answer"/>
+                                            <a t-field="input_line.value_datetime" t-if="question.question_type == 'datetime'"
+                                                class="text-primary filter-add-answer"
+                                                data-model="L" t-att-data-answer-id="input_line.id"
+                                                role="button" aria-label="Filter surveys" title="Only show survey results having selected this answer"/>
+                                            <i class="fa fa-filter text-primary ps-1 filter-add-answer"
+                                                data-model="L" t-att-data-answer-id="input_line.id"
+                                                role="button" aria-label="Filter surveys" title="Only show survey results having selected this answer"/>
```

Comment:
Would something like this work to avoid much repetition?

```suggestion
                                            <a  class="text-primary filter-add-answer"
                                               data-model="L" t-att-data-answer-id="input_line.id"
                                               role="button" aria-label="Filter surveys" title="Only show survey results having selected this answer">
                                                < t-if="question.question_type == 'numerical_box'" t-field="input_line.value_numerical_box" />
                                                < t-elif="question.question_type == 'date'" t-field="input_line.value_date" />
                                                ....
                                                </a>
```